DESCRIPTION

```diff
@@ -24,11 +24,13 @@ Authors@R:
            role="aut", 
            email="infinite.monkeys.with.keyboards@gmail.com", 
            comment = c(ORCID = '0000-0002-3564-4813')))
-Description: Contains a set of modes for iSEE, allowing easy configuration of 
-    iSEE applications.
+Description: iSEEu (the iSEE universe) contains diverse functionality to extend 
+    the usage of the iSEE package, including additional classes for the panels, 
+    or modes allowing easy configuration of iSEE applications.
 License: MIT + file LICENSE
 Encoding: UTF-8
 Depends: 
+    R (>= 4.0.0),
```

Comment:
Fix this.

Reply:
Just to be removed then?
If I recall correctly, sometimes BiocCheck complains to have it specified.

Reply:
Only if you have serialized objects.

Reply:
Ok, gone then ;)